receiver/postgresqlreceiver/scraper.go

```diff
@@ -210,7 +210,13 @@ func (p *postgreSQLScraper) collectQuerySamples(ctx context.Context, dbClient cl
 		return
 	}
 	for _, atts := range attributes {
-		p.lb.RecordDbServerQuerySampleEvent(context.Background(),
+		logCtx := context.Background()
```

Comment:
Same as other question, why use `context.Background` vs. the `ctx` passed in?

Reply:
1. I follow the old code `p.lb.RecordDbServerQuerySampleEvent(context.Background(),` to use `context.Background`.
2. `RecordDbServerQuerySampleEvent` would take the `SpanContext` out of `logCtx`, and fill `trace_id` and `span_id` for logs.